pages.de/common/just.md

```diff
@@ -0,0 +1,7 @@
+# just
+
+> Dieser Befehl ist ein Alias von `just.1`.
```

Comment:
`just` is an alias for two pages. See the english version: https://github.com/tldr-pages/tldr/blob/main/pages/common/just.md